src/supplemental/nanolib/conf_ver2.c

```diff
@@ -381,6 +381,21 @@ conf_basic_parse_ver2(conf *config, cJSON *jso)
 			}
 		}
 	}
+	cJSON *jso_filetransfer = cJSON_GetObjectItem(jso, "filetransfer");
+	if (jso_filetransfer) {
+		conf_filetransfer *filetransfer = &(config->filetransfer);
+
+		hocon_read_bool(filetransfer, enable, jso_filetransfer);
+		char *url = cJSON_GetStringValue(hocon_get_obj("bind", jso_filetransfer));
+		if (url != NULL) {
+			hocon_read_address_base(
+				filetransfer, url, "bind", "mqtt-tcp://", jso_filetransfer);
```

Comment:
bind?

Reply:
Looking forward to a new name, but more importantly I'm still wondering if this parameter is useful to reuse the broker's tcp listener

Reply:
just a name... dont forget the docs